apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -1,17 +1,29 @@
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
 import styles from "./styles.module.css";
 
 type Properties = {
+	imageSrc?: string;
 	name: string;
 };
 
 const FIRST_LETTER_INDEX = 0;
 
-const Avatar = ({ name }: Properties): JSX.Element => {
+const Avatar = ({ imageSrc, name }: Properties): JSX.Element => {
 	const firstLetter = name[FIRST_LETTER_INDEX];
 
 	return (
-		<div className={styles["avatar"]}>
-			<span className={styles["avatar_letter"]}>{firstLetter}</span>
+		<div
+			className={getValidClassNames(
+				styles["avatar"],
+				!imageSrc && styles["no-image"],
```

Comment:
create a separate variable for this and condition below
```suggestion
				!hasImage && styles["no-image"],
```